### DISCUSSION TITLE
Typing overwrites characters rather than inserting new ones

### DISCUSSION THREAD

**Discussion by krzakx** [BryanJenksCommunity/FAQ#163] [Q&A] [answered]
Whenever I open a file (existing one or new one), the cursor automatically switches to Overwrite mode, and this happens only for yours @tallguyjenks vault.
Does it comes from sort of plugin settings?

**Comment by tallguyjenks** [comment, marked as answer]
it might be the vim mode? i might have forgotten to turn that off

**Comment by krzakx** [comment]
Yeap ! That was that.
Thanks
![image](https://user-images.githubusercontent.com/18627339/206672508-6b2374ec-741c-4b02-9398-67acd4cab060.png)
